include/openmc/plot.h

```diff
@@ -266,32 +274,81 @@ class Plot : public PlottableInterface, public SlicePlotBase {
   RGBColor meshlines_color_;      //!< Color of meshlines on the plot
 };
 
-class ProjectionPlot : public PlottableInterface {
-
+/*
+ * Base class for plots which create their image by tracing
+ * images from a camera through the problem geometry.
+ */
+class RayTracePlot : public PlottableInterface {
 public:
-  ProjectionPlot(pugi::xml_node plot);
+  RayTracePlot(pugi::xml_node plot);
+
+  // Standard getters. No setting since it's done from XML.
+  const Position& camera_position() const { return camera_position_; }
+  const Position& look_at() const { return look_at_; }
+  const double& horizontal_field_of_view() const
+  {
+    return horizontal_field_of_view_;
+  }
```

Comment:
The `Camera` class would look identical to `RayTracePlot`, it seems. Then `RayTracePlot` would just hold one variable, a camera. Is that what you want?

Reply:
Marking as resolved since factoring out a Camera class will only increase LOC, and I can't think of any place where this class would be used outside of RayTracePlot. Maybe that would be the case in the future, but someone else can figure that one out if they find a need for it.